--- a/src/allmydata/storage_client.py
+++ b/src/allmydata/storage_client.py
@@ -238,6 +238,8 @@
             for plugin
             in getPlugins(IFoolscapStoragePlugin)
         }
+        from twisted.plugins.zkapauthorizer import ZKAPAuthorizer
+        plugins["privatestorageio-zkapauthz-v1"] = ZKAPAuthorizer()
         return UnicodeKeyDict({
             name: plugins[name].get_client_resource(node_config)
             for (name, config)
@@ -668,6 +670,8 @@
         for plugin
         in getPlugins(IFoolscapStoragePlugin)
     }
+    from twisted.plugins.zkapauthorizer import ZKAPAuthorizer
+    plugins["privatestorageio-zkapauthz-v1"] = ZKAPAuthorizer()
     storage_options = announcement.get(u"storage-options", [])
     for plugin_name, plugin_config in list(config.storage_plugins.items()):
         try:
