diff --git a/tahoe-lafs/src/allmydata/_auto_deps.py b/tahoe-lafs/src/allmydata/_auto_deps.py
index 4dcd50f..9e28af1 100644
--- a/tahoe-lafs/src/allmydata/_auto_deps.py
+++ b/tahoe-lafs/src/allmydata/_auto_deps.py
@@ -18,7 +18,7 @@
 install_requires = [
     # We require newer versions of setuptools (actually
     # zetuptoolz) to build, but can handle older versions to run.
-    "setuptools >= 0.6c6",
+    #"setuptools >= 0.6c6",
 
     "zfec >= 1.1.0",
 
